code/modules/events/vent_clog.dm

```diff
@@ -0,0 +1,116 @@
+/datum/round_event_control/vent_clog
+	name = "Clogged Vents: Normal"
+	typepath = /datum/round_event/vent_clog
+	weight = 10
+	max_occurrences = 3
+	min_players = 10
+
+/datum/round_event/vent_clog
+	announceWhen	= 1
+	startWhen		= 5
+	endWhen			= 35
+	var/interval 	= 2
+	var/list/vents  = list()
```

Comment:
is this list cleared after the event triggers? we don't want to hold hanging vent references